server/Dockerfile

```diff
@@ -16,8 +16,16 @@ RUN apt update \
 
 # install data/make dependencies
 RUN npm install -g mapshaper
+RUN apt update && apt -y install jq
```

Comment:
All of the apt installs can be added to the existing RUN command further up. [Here's a bit more info](https://stackoverflow.com/questions/39223249/multiple-run-vs-single-chained-run-in-dockerfile-what-is-better) on multiple vs single run commands. Not sure how important it ultimately is in this scenario, but at the very least it lets you avoid running apt update twice

Reply:
ah thanks yeah this makes sense! moved it up to the existing one